core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -215,6 +215,10 @@ final case class EitherT[F[_], A, B](run: F[A \/ B]) {
 }
 
 object EitherT extends EitherTInstances {
+  def F[F[_], A, B](a: F[A \/ B]): EitherT[F, A, B] = apply(a)
+  def A[F[_]: Applicative, A, B](a: A): EitherT[F, A, B] = apply(Applicative[F].point(-\/(a)))
+  def B[F[_]: Applicative, A, B](b: B): EitherT[F, A, B] = apply(Applicative[F].point(\/-(b)))
+  def AB[F[_]: Applicative, A, B](d: A \/ B): EitherT[F, A, B] = apply(Applicative[F].point(d))
```

Comment:
Superfluous, because there's `fromDisjucntion`?

Reply:
fromDisjunction is 16 characters... which in combination with the `for` comprehension that it's usually inside is about 25% of the line width.

Reply:
Of these three

```scala
EitherT.AB(d)
EitherT(d.pure[F])
EitherT.fromDisjunction(d)
```

I like the first one the least.

Reply:
hehe, ok but what about something between 2 and 3? How about `EitherT.pure` ?

Reply:
I wouldn't use `pure` for anything other than `B => EitherT[F, A, B]` (`pureRight`), because that's what `pure` from `EitherTMonad` does.